sigopt/magics.py

```diff
@@ -66,8 +66,10 @@ def experiment(self, _, cell):
     experiment_body[PROJECT_KEY] = project_id
     self._experiment = self._connection.experiments().create(**experiment_body)
     print(six.u(
-      'Experiment created, view it on the SigOpt dashboard at https://app.sigopt.com/experiment/{experiment_id}'
-    ).format(experiment_id=self._experiment.id))
+      'Experiment created, view it on the SigOpt dashboard at {api_url}/experiment/{experiment_id}'
```

Comment:
Ah, so, unfortunately this is not the correct URL - we're printing a link to the web (app.sigopt), which is separate from the API (api.sigopt). We have been meaning to work on this, but it's a little odd to stuff our app URL into the API code.

Going to the proposed links will raise errors and not take them to the right paths (it'll say it needs `v1`, and even if we add it, it will complain about not having a `token`)

Reply:
I see, API and WEB are different URLs. But there is still a little bug, and when test local API, app URL should be app.sigopt.ninja

Reply:
Yeah, that's what we'd need to have an overrideable `app_url` for, setting that to `app.sigopt.com` by defualt, but then override to `app.sigopt.ninja` in dev, and `app.staging.sigopt.com`, etc.